network/src/commonTest/kotlin/com/wire/kalium/api/tools/json/api/connection/ConnectionApiTest.kt

```diff
@@ -43,10 +46,30 @@ class ConnectionApiTest : ApiTest {
         connectionApi.fetchSelfUserConnections(pagingState = null)
     }
 
+    @Test
+    fun givenACreationRequest_whenRequestingAConnectionWithAnUser_thenShouldReturnsACorrectConnectionResponse() = runTest {
+        val httpClient = mockAuthenticatedHttpClient(
+            CREATE_CONNECTION_RESPONSE.rawJson,
+            statusCode = HttpStatusCode.OK,
+            assertion = {
+                assertJson()
+                assertPost()
+                assertPathEqual("$PATH_CONNECTIONS_ENDPOINT/domain_id/user_id")
+            }
+        )
+
+        val connectionApi = ConnectionApiImpl(httpClient)
+        val response = connectionApi.createConnection(UserId("user_id", "domain_id"))
```

Comment:
What about instead of hardcoding these _user_id_ and _domain_id_, put these into dummy variables and pass them over to the assertion and the creation of the `UserId` object? that way it'll be easier and cleaner to modify these values in the future for this method, or following ones that get inspired with it?

Reply:
Not really a blocker for these small tests but maybe also add some `// Given  // When  // Then` comments to keep the consistence with others.

Reply:
Done! changes applied =)